src/viewProviders/topics.ts

```diff
@@ -290,12 +294,65 @@ export class TopicViewProvider implements vscode.TreeDataProvider<TopicViewProvi
       },
     );
 
+    // A schema subject was added or removed.
+    const schemaSubjectChangedSub: vscode.Disposable = schemaSubjectChanged.event(
+      (event: SubjectChangeEvent) => {
```

Comment:
This (and `schemaVersionsChangedSub`) are super nice for the Topics view. Could definitely see a small follow-on branch to hook in the same event firing from schema upload as well since right now that's a manual-refresh action (for the Topics view only; Schemas view refreshes automatically). 💯

Reply:
Is the plan! However schema upload does the refreshes of the views is inferior to this mechanism.